administration/src/cards/CsvFactory.ts

```diff
@@ -37,8 +36,6 @@ export function generateCsv(
 
 export const getCSVFilename = (cardBlueprints: CardBlueprint[]): string => {
   const filename =
-    cardBlueprints.length === 1
-      ? findExtension(cardBlueprints[0].extensions, NuernbergPassIdExtension)?.state?.passId
-      : 'Pass-ID[0]mass'
```

Comment:
This only works for nuernberg, does it? What is the file name for other projects?

Reply:
we don't have csv export for other projects, so no filename needed or what do you mean?

Reply:
Ahh thanks for the explanation :+1:

Reply:
but actually this fallback is ugly i would use sth like `SozialpassMassExport`